docs/providers/openwhisk/cli-reference/create.md

```diff
@@ -62,12 +63,9 @@ The provider which is used for deployment later on is Apache OpenWhisk.
 serverless create --template openwhisk-nodejs --path my-new-service
 ```
 
-This example will generate scaffolding for a service with `openwhisk` as a provider and `nodejs` as runtime. The scaffolding
-will be generated in the `my-new-service` directory. This directory will be created if not present. Otherwise Serverless
-will use the already present directory.
+This example will generate scaffolding for a service with `openwhisk` as a provider and `nodejs` as runtime. The scaffolding will be generated in the `my-new-service` directory. This directory will be created if not present. Otherwise Serverless will use the already present directory.
```

Comment:
Either works, you can just use `nodejs` for the default version or add an explicit version tag.